server/belga/io/feed_parsers/base_belga_newsml_1_2.py

```diff
@@ -279,9 +307,13 @@ def parser_newsmanagement(self, item, manage_el):
         if manage_el is None:
             return
 
-        element = manage_el.find('NewsItemType')
-        if element is not None:
-            item['news_item_type'] = element.get('FormalName', '')
+        for element in manage_el.findall('NewsItemType'):
+            if element is not None:
+                item.setdefault('subject', []).append({
+                    "name": element.get('FormalName'),
+                    "qcode": element.get('FormalName'),
+                    "scheme": "news_item_type"
+                })
 
         element = manage_el.find('FirstCreated')
         if element is not None:
```

Comment:
We must convert `firstcreated` and `versioncreated` to utc. See https://github.com/superdesk/superdesk-belga/blob/master/server/belga/io/feed_parsers/belga_newsml_1_2.py#L268
UPD: I see that we convert to UTC later in the not base classes, but what about `BelgaEFENewsMLOneFeedParser` ?

Reply:
Because I see its 'Datetime' format already have timezone is UTC(**+0000**), EFE (20190121T103600 **+0000**), AFP ( 20190121T104233 **+0000**) , So I don't convert it. 
ANP (20181210T123731 **+0100**), it have time zone(**+0100**), so I convert it to utc. 
TASS (20190121T102708), it haven't timezone, maybe It use local time. I check all ingest files, all of them have location at "MOSCOW", So I think its local timezone in Moscow, And I set it is "Moscow" timezone and convert it to UTC (like Belga).